src/formio/components/AddressNL.js

```diff
@@ -162,6 +171,19 @@ export default class AddressNL extends Field {
     changed && this.renderReact();
     return changed;
   }
+
+  // Called in the representation of an edit grid
+  getValueAsString(value, options) {
+    if (!value || !Object.keys(value).length) return '';
+    const {postcode, houseNumber, houseLetter, houseNumberAddition, streetName, city} = value;
+    return `
+        <address>
+          ${postcode} ${houseNumber}${houseLetter} ${houseNumberAddition}
+          <br>
+          ${streetName || ''} ${city || ''}
```

Comment:
I don't think the linebreak should be there if there is no streetname and no city